app-misc/lirc/lirc-0.9.4a-r2.ebuild

```diff
@@ -22,7 +22,7 @@ fi
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
-IUSE="doc static-libs X audio irman ftdi inputlirc iguanair systemd usb"
```

Comment:
any specific reason why you removed static-libs? Not that I mind it in any way, but was it causing issues?